### ISSUE TITLE
[booth] unhandled (or unguarded) specification in `pcs booth create ip IP/MASK`

### ISSUE THREAD

**Issue by jnpkrn** [ClusterLabs/pcs#220]
Only `IP` is supported, owing to what `ocf:heartbeat:IPaddr2` supports
behind `ip` parameter.

```
 # pcs resource show booth-booth-ip
 Resource: booth-booth-ip (class=ocf provider=heartbeat type=IPaddr2)
  Attributes: ip=10.37.164.25/22
  Operations: monitor interval=10s timeout=20s (booth-booth-ip-monitor-interval-10s)
              start interval=0s timeout=20s (booth-booth-ip-start-interval-0s)
              stop interval=0s timeout=20s (booth-booth-ip-stop-interval-0s
```

```
# pcs resource debug-start booth-booth-ip --full
> [...]
>  stderr: ++ 17:54:20: findif_check_params:170: ipcheck_ipv4 10.37.164.25/22
>  stderr: ++ 17:54:20: ipcheck_ipv4:3: local 'r1_to_255=([1-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])'
>  stderr: ++ 17:54:20: ipcheck_ipv4:4: local 'r0_to_255=([0-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])'
>  stderr: ++ 17:54:20: ipcheck_ipv4:5: local 'r_ipv4=^([1-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.([0-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.([0-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.([0-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$'
>  stderr: ++ 17:54:20: ipcheck_ipv4:6: grep -q -Ee '^([1-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.([0-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.([0-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.([0-9][0-9]?|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$'
>  stderr: ++ 17:54:20: ipcheck_ipv4:6: echo 10.37.164.25/22
>  stderr: ++ 17:54:20: findif_check_params:171: '[' 1 = 1 ']'
>  stderr: ++ 17:54:20: findif_check_params:172: ocf_log err 'IP address [10.37.164.25/22] not valid.'
> [...]
```

Handling this correctly preferred over crippling the expressiveness.

**Comment by tomjelinek** [comment]
Ack, tracked at https://bugzilla.redhat.com/show_bug.cgi?id=1791601

**Comment by tomjelinek** [comment]
This is partially resolved by #814 and will be resolved completely once agent validation gets to be enabled non-optionally.
